service/templates/projects.html

```diff
@@ -275,6 +275,14 @@ <h3> Admins </h3>
           <div class="muted small">When checked, tasks from previous revisions will be unscheduled when the equivalent task in a newer commit finishes successfully.</div>
         </div>
       </div>
+      <div class="form-group">
```

Comment:
should we create a ticket to add this to the new project page?

Reply:
Created https://jira.mongodb.org/browse/EVG-16360 already but I'm not sure if it actually belongs on the project page going forward or if something on mainline would be more intuitive.